toolchain.cmake

```diff
@@ -112,6 +113,17 @@ elseif(MICROCHIP_MCU MATCHES "^(dsPIC|PIC)(32M[XZ]|[0-9]+[A-Z])([A-Z0-9]+)$")
             "Unsupported MCU family '${MICROCHIP_MCU_FAMILY}'."
         )
     endif()
+    
+elseif(MICROCHIP_MCU MATCHES "^(AT)(tiny|xmega)([a-zA-Z0-9]+)$")
+    set(MICROCHIP_MCU_FAMILY "${CMAKE_MATCH_1}${CMAKE_MATCH_2}")
+    set(MICROCHIP_MCU_MODEL  "${MICROCHIP_MCU}")
+    if(MICROCHIP_MCU_FAMILY IN_LIST MICROCHIP_FAMILIES_8)
```

Comment:
Building a test program to check if the compiler actually supports the CPU would make error messages nicer - i.e. a descriptive error during configure instead of during compile (though still descriptive).

Reply:
This idea sounds good. but as above: feel free :)
I can try to manage a configuration test, but i will need some time for it. I want to use this toolchain for some of our components and I just want to share my adaptions, which I made to fit my purposes, so a configuration test is currently not on my way :)